cmd/internal/core/reconfigure-switch.go

```diff
@@ -133,7 +133,7 @@ func (c *Core) buildSwitcherConfig(s *models.V1SwitchResponse) (*switcher.Conf,
 		if v, has := p.Vrfs[nic.Vrf]; has {
 			vrf = v
 		}
-		vni64, err := strconv.ParseUint(strings.TrimPrefix(nic.Vrf, "vrf"), 10, 32)
+		vni64, err := strconv.ParseUint(strings.TrimPrefix(nic.Vrf, "Vrf"), 10, 32)
```

Comment:
this is breaking for existing installations, could we instead parse and Camelcase the vrf in the sonic impl. only ?

Reply:
I don't understand why the `metal-api` has to define the name of the Vrfs. The name is a local property and not a global one.